### ISSUE TITLE
Mongodb null field not support in 0.28

### ISSUE THREAD

**Issue by dingziran** [metabase/metabase#6880]
When I update metabase to 0.28, it cannot scan any table which has null value.

The error message is

```
Feb 06 20:00:06 ERROR metabase.sync.util :: Error syncing fields for Table 163 'application_analyze': Input to most-common-object-type does not match schema: 

 [(named [[(named (not (instance? java.lang.Class nil)) "Class") nil]] field-types)] 
```

I install metabase by docker. And earlier I am using 0.27, everything is ok. I update metabase and using my old metabase.db file.

**Comment by camsaul** [comment]
Thanks @dingziran I will take a look at this.

**Comment by salsakran** [comment]
closed via #6881

**Comment by aviggiano** [comment]
I believe this is related to #6975
